config/example/cli.yaml

```diff
@@ -0,0 +1,5 @@
+wallet: ./wallets/wallet.json               # path to the NEP-6 wallet file or path to private key file or WIF
```

Comment:
begin file with `---\n\n`

`./` is required for execution; files could be specified without it: `wallets/wallet.json`